# Project Analysis Proposal: Characterizing Opsin Protein Spatial Expression and Ommatidia Morphology in Speyeria mormonia Butterflies: Insights into Function and Visual Ecology

**By:** Sofia Diaz 

## Introduction

The evolution of visual systems is central to evolutionary biology due to its contribution to species diversification. Investigations into color vision and visual systems have been instrumental in understanding how organisms perceive their environments and interact with one another (Gerl and Morris 2008). Unique visual genes, such as opsins, are responsible for detecting specific wavelengths of light and have been observed in various organisms (Terakita 2005). Located in retinal photoreceptor cells, opsins enable light detection, triggering phototransduction, which allows organisms to interpret visual cues (Briscoe and Chittka 2001). Studies have shown that gene duplications have led to the diversity of opsins, resulting in variation in photoreceptor types and visual abilities, and consequently, differences in behaviors in response to diversified opsin patterning across species (Briscoe and Chittka 2001; Frentiu et al. 2007; Feuda et al. 2016). Additionally, exploring the structure and morphology of the eye, such as ommatidia structure, provides a link to eye morphology and functionality. In butterflies, color vision is critical for behaviors like foraging, mate selection, and ovipositing (Arikawa 2017). The compound eye of a butterfly is composed of many structural units called ommatidia; each individual ommatidium contains a cluster of nine photoreceptor cells, exhibiting opsin configurations that lead to species-specific visual abilities (Briscoe and Chittka 2001). Nymphalid butterflies are a diverse family of brush-footed butterflies that have been known to possess limited photoreceptor diversity with typically three to four spectral types of photoreceptors (McCulloch et al. 2016). The spatial expression of opsin proteins has been documented in some nymphalid butterfly species, highlighting opsin patterning of the R1 through R8 photoreceptor cells (McCulloch et al. 2016). To further understand the extent of photoreceptor diversity, color vision, and its link to function in Nymphalidae, additional nymphalid butterflies must be studied. In this project, I examined *Speyeria mormonia*, a nymphalid butterfly species from Colorado. *Speyeria mormonia* opsin expression has yet to be characterized at the protein level, and ommatidia morphology has yet to be documented. This study aims to characterize opsin protein expression and ommatidial morphology in male *Speyeria mormonia* butterfly eyes to explore opsin configurations and ommatidial structure in relation to their functions and visual ecology in their natural environment.

The proposed project is well suited for our bioinformatics course, as tools such as RStudio will be used to explore *Speyeria mormonia* opsin expression and ommatidia morphology. By quantifying these datasets, bioinformatics can statistically test opsin pattern types and ommatidial structure types to identify potential differences and similarities within the male *Speyeria mormonia* eyes. These analyses will reveal what patterns are present, in what quantities, and whether these patterns and morphologies are statistically significant, demonstrating their functional and behavioral relevance. Bioinformatics will therefore enhance the visualization and characterization of opsin spatial expression and ommatidia morphology, deepening our understanding of color vision in the *Speyeria mormonia* system.

## Identify how the proposed analyses pertain to the student’s research goals.

This proposed analysis aligns with my dissertation research goals, as I aim to characterize opsin spatial expression in multiple nymphalid butterfly species to better understand their color vision capabilities. To assess how nymphalids perceive color and visual cues, it is essential to examine the protein expression of opsins within the ommatidia of their eyes, identifying which opsin types are present in the photoreceptors, their quantities, and their distribution throughout the eye. In addition to opsin spatial expression, analyzing the overall structure of the ommatidia is informative for understanding visual function. By studying both protein expression and eye structure together, I hope to determine what is present, in what quantities, and how these differences enhance our understanding of nymphalid life history and function.

This analysis uses my dataset from Speyeria mormonia butterflies, where I am characterizing opsin pattern types in the ommatidia, quantifying opsin expression by pattern type, and examining dorsal and ventral differences to infer how male Speyeria mormonia perceive color. These findings can then be linked to their behaviors and functions. Because this is exploratory work, examining both the protein and morphological aspects of ommatidia may reveal unique visual capabilities and provide insights into natural behaviors such as mating, oviposition, and host-plant identification. This research will help clarify how color vision relates to these life-history behaviors and the butterflies’ interaction with visual cues in their natural environments.

Overall, my research explores the connection between color vision and function in nymphalid butterflies. By examining opsin protein expression and eye structure, I aim to infer how visual abilities shape behavior and functioning. This project summarizes my approach to investigating this link for my dissertation.

## Identify specific dataset(s) and how the student will acquire them.

During the summer, I conducted immunohistochemistry (IHC) experiments on male Speyeria mormonia butterflies, generating a large dataset of confocal images to examine opsin protein expression in the R1/R2 photoreceptor cells. For my proposed analysis, I will use two IHC datasets. The first is an ommatidia opsin count dataset, focusing on the opsin proteins present within the different photoreceptor cells in the ommatidia. I will manually count the opsin pattern types of the R1/R2 photoreceptor cells from six high-quality cross-sectional images, ensuring the tissue is intact and cell bodies are clearly labeled with minimal background. Each unique pattern will be visually marked throughout the image. The second dataset will quantify ommatidia morphology by identifying and counting the types of ommatidia shapes in six of the best longitudinal IHC images, taken to explore the structure and shape of the ommatidia in male *Speyeria mormonia* eyes. For both datasets, I will input the quantitative data into an Excel spreadsheet for easy transport and upload into RStudio for analysis.

## Identify specific analyses they will conduct on the data.
To generate my two datasets, I will use ZEISS ZEN Lite (ZEISSLITE), the lite version of Zen Blue, which I used to conduct all the imaging for my *Speyeria mormonia* samples. This software lets me view confocal images at high resolution, so I can clearly see all opsin pattern types and ommatidia morphologies for marking and generating quantitative data. Additionally, I will use ZEISSLITE to capture representative images of each opsin pattern and ommatidia morphology type, which I will use to create a microscopy panel figure of the different types and combinations observed in my dataset.

For my data analysis plan, I will count ommatidial morphologies and photoreceptor subtypes in all images and group them by pattern and morphological type. I will convert these counts into proportions to quantify each type. For both datasets—the ommatidia morphology and the opsin pattern types—I will create bar plots in RStudio to visualize the distribution of photoreceptor subtypes and ommatidia morphologies within the eyes of male *Speyeria mormonia* butterflies. Bar plots will allow for the comparison of the abundance of different photoreceptor types and ommatidia morphologies. In addition to bar plots, I plan to perform principal coordinate analysis (PCoA) to examine similarities and differences in opsin patterning and ommatidia morphologies across the six male butterflies. I also plan to use a Chi-squared goodness-of-fit test to determine whether the proportions of photoreceptor types and ommatidia morphologies show significant patterns. Finally, I will use RStudio to generate microscopy panels of my IHC confocal images to visually display photoreceptor pattern types (based on UV and blue opsin expression) and the various ommatidia morphologies found throughout the eye.

## **Identifying Specific Visualizations they will make regarding the data.**  
The visualizations I plan to create for my two datasets will show UV and blue opsin expression in the R1/R2 photoreceptor cells within the ommatidia of six male *Speyeria mormonia* eyes. My goal is to display how opsin expression appears in these cells, identify any patterns in opsin presence, and quantify these patterns to determine if consistent opsin patterning exists in the male *Speyeria mormonia* system. Additionally, I will visualize the structure of individual ommatidial units to better understand the morphology of the male eye and how its components contribute to color vision. By analyzing both datasets, I aim to explore how color vision is connected to functionality and behavior. Protein expression reveals what happens inside each ommatidium, while ommatidial morphology examines the structure of all ommatidia as a whole. Studying both the opsin expression in individual ommatidia and the overall morphology will aid in the better understanding of the visual capabilities in *Speyeria mormonia* butterflies.

## Make a persuasive case that these analyses are feasible.
The analyses I propose for my project are entirely feasible within the scope of the class. I will be working with smaller datasets, which makes it manageable to create both the plots and microscopy figures. I plan to generate straightforward bar plots and principal component analyses to visualize the opsin and ommatidia data, highlighting quantitative outcomes for photoreceptor types and ommatidia shapes and structures. This will provide a clear visual representation of the abundance of each pattern and morphological type for comparison. I will use the Chi-Squared test to statistically determine if the observed patterns in the photoreceptors and ommatidia are significant. Additionally, I will create microscopy panels for each dataset to showcase the different photoreceptor types and ommatidial morphologies. Overall, I will generate three main figures per dataset: a bar plot, a principal coordinate analysis, and a microscopy panel, along with Chi-Squared test results. This totals six figures for the entire proposal, which is achievable within the class timeframe and will allow me to successfully explore the connections between color vision, functionality, and behavior.

## Conclusion
By examining opsin protein expression and eye morphology, we can identify how species perceive color and their range of color vision. These insights will aid in understanding visual systems and their adaptations to various environments. The spatial expression of opsin proteins in *Speyeria mormonia* butterflies remains an unexplored frontier, representing a critical gap in our understanding of their visual system. By analyzing ommatidia structural types and opsin patterns, this research not only sheds light on the intricate mechanisms underpinning color vision in *Speyeria mormonia* but also highlights how unique cellular morphologies and protein expression drive distinct behavioral and ecological outcomes. This project aims to connect opsin expression and morphology with color vision, behavior, and ecological function in this butterfly species to gain a further understanding of the extent of color vision within the family Nymphalidae. And utilizing bioinformatics will effectively communicate the opsin spatial expression and ommatidia data, showcasing valuable insights that could transform our broader understanding of butterfly visual ecology and evolutionary biology.

## **References**

Arikawa K, Scholten DG, Kinoshita M, Stavenga DG. 2017\. The eyes and vision of butterflies. J Physiol 595: 5457-5464.

Briscoe AD, Chittka L. 2001\. The evolution of color vision in insects. Annu Rev Entomol 46: 471-510.

Feuda R, Hamilton SC, McInerney JO, Pisani D. 2016\. Conservation, duplication, and divergence of five opsin genes in insect evolution. Genome Biol Evol 8: 2882-2898.

Frentiu CD, Yuan F, Savage WK, Bernard GD, Mullen SP, Briscoe AD. 2007\. Adaptive evolution of color vision as seen through butterflies. Proc Natl Acad Sci USA 104: 8634-8640.

Gerl EJ, Morris MR. 2008\. The causes and consequences of color vision. Evo Edu Outreach 1: 476-486.

McCulloch KJ, Osorio D, Briscoe AD. 2016\. Sexual dimorphism in the compound eye of Heliconius erato. J Exp Biol 219: 3009-3017. Terakita A. 2005\. The opsins. Genome Biol 6: 213\. doi:10.1186/gb-2005-6-3-213
